### ISSUE TITLE
Important icons/indicators are too small, easy to miss.

### ISSUE THREAD

**Issue by jonathansampson** [brave/brave-browser#2657]
## Description 
Users are able to easily miss important notices and icons which can directly impact their experience.

## Actual result:
Important notices are easy to miss, or easy to misunderstand. (See screenshots)

## Expected result:
Users should not be able to easily miss (or misunderstand) important notices.

## Brave version (brave://version info)
0.57.18

### Reproducible on current release:
Yes

#### Widevine

When visiting a site like Netflix or Hulu, the user will see a brief fly-out at the right-side of the address bar. The message doesn't mention the site, or even that Widevine is _required_ to view the site's media. After a moment, the message collapses into a small puzzle icon.

The user can click the puzzle icon to reveal 107 words that also do not media video or audio. Again, no indication is given that the present site _requires_ Widevine in order to function properly.

![image](https://user-images.githubusercontent.com/815158/50247082-90c32380-039c-11e9-93e5-9d4a457838d4.png)

#### Autoplay

When visiting a site like Netflix or CWTV.com, the user will see a brief fly-out at the right-side of the address bar. The message states that autoplay content has been prevented. After a moment, the message collapses into a small play-button icon.

The user can click the play-button icon and choose to _Always enable autoplay_. Often the user will need to refresh the page or re-engage media controls to see results, but this is not immediately clear to the user.

![image](https://user-images.githubusercontent.com/815158/50247113-a46e8a00-039c-11e9-916c-db515340381d.png)

#### Geolocation

![image](https://user-images.githubusercontent.com/815158/50247148-b2240f80-039c-11e9-82c7-d7ef3b806145.png)

#### User Media

![image](https://user-images.githubusercontent.com/815158/50247176-c831d000-039c-11e9-9866-6f5209626e50.png)

**Comment by kjozwiak** [comment]
re: Netflix, there's a discussion happening in https://github.com/brave/brave-browser/issues/1940 to make the notification more visible/easier to see for users.

**Comment by rossmoody** [comment]
Going to tackle a few of these considerations in an upcoming autoplay/notification revamp but to make this issue more actionable we might want to break it apart. Tough to have a single PR or even multiple that addresses all 4 of these silo'd cases.

**Comment by kjozwiak** [comment]
@rossmoody agreed, sometimes we tend to miss things in general when there's multiple bugs/problems in a single issue and only `x` and `y` get resolved but `z` doesn't.

When you create the issues, can you link them in here and close this off? So there's at least a record/trial someone can follow if they run into this issue in the future while searching etc..

**Comment by bsclifton** [comment]
This issue hasn't had an update in a while - it's met the criteria for being stale. The issue is going to be closed for now. If this is still a problem, please leave a comment or re-open the issue. Thanks!
